ui/revenuecatui/build.gradle

```diff
@@ -53,6 +54,7 @@ dependencies {
     implementation libs.coil.svg
     debugImplementation libs.compose.ui.tooling
     debugImplementation libs.androidx.test.compose.manifest
+    implementation libs.kotlinx.serialization.json
```

Comment:
Hmm should we add this to the test target only? `testImplementation`. I think we're not serializing in this module other than in tests

Reply:
oh yes! good call